modules/gdnative/gdnative_api.json

```diff
@@ -1,22 +1,22 @@
 {
   "core": {
-    "type": "CORE",
-    "version": {
-      "major": 1,
-      "minor": 0
-    },
-    "next": {
       "type": "CORE",
       "version": {
         "major": 1,
-        "minor": 1
+        "minor": 0
```

Comment:
You're still changing the indentation, causing a diff while no actual change is made to the `core` definitions.

Reply:
@akien-mga I see the issue now, it was not showing up in my editor very clearly. I need to increase tab size for json... it has been fixed in the latest force push.